src/GovUk.Education.ExploreEducationStatistics.Admin/Security/AuthorizationHandlers/UpdateContactAuthorizationHandler.cs

```diff
@@ -7,37 +7,37 @@
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.Security.AuthorizationHandlers
 {
-    public class UpdatePublicationRequirement : IAuthorizationRequirement
+    public class UpdateContactRequirement : IAuthorizationRequirement
     {
     }
 
-    public class UpdatePublicationAuthorizationHandler : AuthorizationHandler<UpdatePublicationRequirement, Publication>
+    public class UpdateContactAuthorizationHandler : AuthorizationHandler<UpdateContactRequirement, Publication>
     {
         private readonly AuthorizationHandlerResourceRoleService _authorizationHandlerResourceRoleService;
 
-        public UpdatePublicationAuthorizationHandler(
+        public UpdateContactAuthorizationHandler(
             AuthorizationHandlerResourceRoleService authorizationHandlerResourceRoleService)
         {
             _authorizationHandlerResourceRoleService = authorizationHandlerResourceRoleService;
         }
 
         protected override async Task HandleRequirementAsync(AuthorizationHandlerContext context,
-            UpdatePublicationRequirement requirement,
+            UpdateContactRequirement contactRequirement,
             Publication publication)
         {
             if (SecurityUtils.HasClaim(context.User, UpdateAllPublications))
             {
-                context.Succeed(requirement);
+                context.Succeed(contactRequirement);
                 return;
             }
-                        
+
             if (await _authorizationHandlerResourceRoleService
                     .HasRolesOnPublication(
                         context.User.GetUserId(),
                         publication.Id,
                         Owner))
             {
-                context.Succeed(requirement);
+                context.Succeed(contactRequirement);
```

Comment:
Description of this PR says that only BAU can manage the publication details so would expect there to be nothing here about the Publication Owner role. 

Also if this is the case, then rather than having an auth handler which checks the `UpdateAllPublications` claim, you could just do one of these two options:


```
options.AddPolicy(SecurityPolicies.CanUpdateContact.ToString(), policy => policy.RequireRole(RoleNames.BauUser));
```

or

```
options.AddPolicy(SecurityPolicies.CanUpdateContact.ToString(), policy => policy.RequireClaim(SecurityClaimTypes.UpdateAllPublications.ToString()));
```

No auth handler required! 🙂

Reply:
> Description of this PR says that only BAU can manage the publication details so would expect there to be nothing here about the Publication Owner role.

The intention was for BAU users to be able to change all publication details, and publication owners to only be able to change the publication summary. I've updated the description to remove any ambiguity.